### ISSUE TITLE
CheckBoxListItem does not update aria-checked selected

### ISSUE THREAD

**Issue by pfesenmeier** [radzenhq/radzen-blazor#1653]
**Describe the bug**
CheckBoxListItem does not update aria-checked when selected

**To Reproduce**
1. Go to https://blazor.radzen.com/checkboxlist
2. Open up markup in devtools. Scroll to div with class "rz-helper-hidden-accessible", find the input inside
3. Select checkbox
4. See that input's aria-checked remains false regardless of selection

**Expected behavior**
aria-checked should be true / false based on selection
**Screenshots**
![image](https://github.com/user-attachments/assets/504972db-6dca-484f-ad3e-26a4b2fa9cbc)

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser Edge
 - Version 127.0.2651.98
